src/gpu/nvidia/cudnn_binary.cpp

```diff
@@ -19,6 +19,9 @@
 #include "gpu/nvidia/sycl_cuda_scoped_context.hpp"
 #include "gpu/nvidia/sycl_cuda_stream.hpp"
 #include "sycl/sycl_buffer_memory_storage.hpp"
+#include "sycl_cuda_helper.hpp"
+
+#include <optional>
```

Comment:
Looks like this optional belongs to a different file, not this one.

Reply:
fixed